docs/content/temporary-components.md

```diff
@@ -0,0 +1,43 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../README.md#recommended-use).
+
+-----
+
+
+## Temporary components
+
+The attribute `Improbable.Gdk.Core.RemoveAtEndOfTick`, can be applied to any ECS component, extending either `IComponentData` or `ISharedComponentData`.
+All components with this attribute will be removed from all entities during the `InternalSpatialOSCleanGroup` at the end of `SpatialOSSendGroup`.
```

Comment:
do we have a ticket to update the doc after we added the documentation about the groups?

Reply:
I think this is consistent with the current implementation of update groups. We just haven't written the doc explaining exactly what these are yet 😬

Reply:
it's more about adding a reference to that doc as soon as it exists ;)

Reply:
There is a ticket for that doc to exist.
There is another one to reorganise the docs completely too.